app/src/main/java/com/haigcramlin/cars/MainActivity.kt

```diff
@@ -1,31 +1,144 @@
 package com.haigcramlin.cars
 
-import android.content.Context
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
-import android.util.AttributeSet
 import android.view.View
-import android.widget.Button
-import android.widget.TextView
-import android.widget.Toast
+import android.widget.*
 import kotlinx.android.synthetic.main.activity_main.*
 
 class MainActivity : AppCompatActivity() {
+    var car1: String? = null
+    var car2: String? = null
+    var winnerImage: ImageView? = null
+    var trophyImage: ImageView? = null
+    var trophyImage2: ImageView? = null
+    var removeCarImage: ImageView? = null
+    var someCars: List<String?> = arrayListOf(car1, car2)
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
-        // FTW
+        trophyImage = findViewById(R.id.blank_trophy)
```

Comment:
This is fine, but Kotlin also allows us to just use the id of the element in code. So also acceptable is not using find by id to populate an otherwise nullable var and using blank_trophy in the code instead. 

Look up Kotlin synthetic variables. There is also some talk that using them is not recommended. https://proandroiddev.com/the-argument-over-kotlin-synthetics-735305dd4ed0
But they are easy and at this point using it is personal pref

Reply:
good to know. Will read.